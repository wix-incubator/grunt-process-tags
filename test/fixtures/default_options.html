<!-- process_statics prefix('first/') -->
<script src="http://source.with.scheme/should/not/be/replaced"></script>
<script src="should/be/prefixed/by/first"></script>
<!-- end_process_statics -->

<!-- process_statics prefix('second/') -->
<script src="should/be/prefixed/by/second"></script>
<!-- end_process_statics -->

<!-- process_statics prefix('third/') -->
<script src="third/should/not/be/prefixed/by/third/again"></script>
<!-- end_process_statics -->

<!-- process_statics prefix('empty/block/should/be/removed') -->
<!-- end_process_statics -->
