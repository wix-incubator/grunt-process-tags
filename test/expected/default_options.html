<script src="http://source.with.scheme/should/not/be/replaced"></script>
<script src="first/should/be/prefixed/by/first"></script>

<script src="second/should/be/prefixed/by/second"></script>

<script src="third/should/not/be/prefixed/by/third/again"></script>


